gatsby-config.js

```diff
@@ -80,6 +82,14 @@ module.exports = {
       },
     },
     `gatsby-plugin-catch-links`,
+    {
+      resolve: "gatsby-plugin-root-import",
```

Comment:
This allows absolute references in MD rather than relative
e.g. `/news` for the news page from anywhere instead of `../../news`, for example